### PR TITLE
Added ImportTransformer

### LINKED ISSUE #1084: Missing test cases for function/const imports
Afaik we do not have any test case for these imports:

```php
use function Foo\bar;
use const Foo\BAZ;
```

The "function" and "const" parts are tokenized as `T_FUNCTION` and `T_CONST`, so they may break fixers using these tokens.
And we should check `UnusedUseFixer` and `OrderedUseFixer`.

### PR DESCRIPTION
**Pull request by gharlan**
resolves #1084

### CODE REVIEW

File: Symfony/CS/Tokenizer/Transformer/ImportTransformer.php [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+ */
+
+namespace Symfony\CS\Tokenizer\Transformer;
+
+use Symfony\CS\Tokenizer\AbstractTransformer;
+use Symfony\CS\Tokenizer\Token;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * Transform T_CONST/T_FUNCTION into CT_CONST_IMPORT/CT_FUNCTION_IMPORT.
```
Comment (keradus) [inline Symfony/CS/Tokenizer/Transformer/ImportTransformer.php]: Looking on descr it will transform `function foo ()  {}` too?

**Comment by keradus** [comment]
:+1:

**Comment by GrahamCampbell** [comment]
:+1:

**Comment by keradus** [comment]
Thank you @gharlan.

**Pull request merged**